src/lib/components/Modal.svelte

```diff
@@ -30,8 +30,16 @@
 
   const modalTitleId = nextElementId("modal-title-");
   const modalContentId = nextElementId("modal-content-");
+
+  const handleKeyDown = (event: KeyboardEvent) => {
```

Comment:
I once read it was useful to not name event as such because once bundle it might conflicts with event if polyfied. Long story short, `{key}`?